CHANGELOG.md

```diff
@@ -160,6 +160,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * (x/distribution) [#17115](https://github.com/cosmos/cosmos-sdk/pull/17115) Migrate `PreviousProposer` to collections.
 * (x/upgrade) [#16244](https://github.com/cosmos/cosmos-sdk/pull/16244) upgrade module no longer stores the app version but gets and sets the app version stored in the `ParamStore` of baseapp.
+* (x/staking) [#17655](https://github.com/cosmos/cosmos-sdk/pull/17655) `HistoricalInfo` was replaced with `Historical`, it removes the validator set and comet header and only keep what is needed for IBC. 
```

Comment:
I feel like the changelog does not correspond to the latest updates in this PR.

Reply:
Can we add a Client Breaking change as well, given that the response will be different?